### PR TITLE
use a TypeVar for asyncio.BaseProtocol

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: uvloop/loop.pyi [hunk 1]
```diff
@@ -21,7 +21,7 @@ _T = TypeVar('_T')
 _Context = Dict[str, Any]
 _ExceptionHandler = Callable[[asyncio.AbstractEventLoop, _Context], Any]
 _SSLContext = Union[bool, None, ssl.SSLContext]
-_TransProtPair = Tuple[asyncio.transports.BaseTransport, asyncio.protocols.BaseProtocol]
+_ProtocolT = TypeVar("_ProtocolT", bound=asyncio.BaseProtocol)
```
Comment (graingert) [inline uvloop/loop.pyi:24]: this follows the implementation from https://github.com/python/typeshed/pull/6937/files#diff-4886af916d5482ca50a0cfea90fe0d403cc5a98e1087598a14d12c9370cfb6b4R19

**Review by fantix — APPROVED** [review]
Thank you for the quick correction!

**Comment by graingert** [comment]
> Thank you for the quick correction!

I had this as a PR already - I think it was just missed

**Comment by fantix** [comment]
> > Thank you for the quick correction!
> 
> I had this as a PR already - I think it was just missed

Right that was on me - shouldn't have reviewed from newest.

**Pull request merged**